server/src/test/java/org/opensearch/cluster/routing/MovePrimaryFirstTests.java

```diff
@@ -113,6 +120,6 @@ public void testClusterGreenAfterPartialRelocation() throws InterruptedException
             internalCluster().stopRandomNode(InternalTestCluster.nameFilter(z1n1));
             internalCluster().stopRandomNode(InternalTestCluster.nameFilter(z1n2));
```

Comment:
Do we have to shutdown nodes z2n1 and z2n2 as well here?

Reply:
There are 4 nodes in the cluster. If we shutdown all 4, cluster will not be green. We want to shutdown all excluded nodes (in this case 2) after 60% of total shards have relocated to z2n1 and z2n2. Due to [#1445 ] all primaries would have started in those 60% and hence, cluster will become eventually green